scripts/verify.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * WayFinder
+ * Copyright (C) 2022-2025 Permanent Data Solutions, Inc. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
```

Comment:
Side note: we should probably MIT or Apache license this (whatever we do on SDKs).

Reply:
updated here [6d42820](https://github.com/ar-io/wayfinder/pull/26/commits/6d42820c7ca84a3c033903f6de7ee5bc221f18f9)[6d42820](https://github.com/ar-io/wayfinder/pull/26/commits/6d42820c7ca84a3c033903f6de7ee5bc221f18f9)